pkg/server/webserver.go

```diff
@@ -30,7 +30,7 @@ func StartWebServer(addr string, tlsConfig *tls.Config, s schema.ImmuServiceServ
 
 	go func() {
 		var err error
-		if tlsConfig != nil {
+		if len(tlsConfig.Certificates) > 0 {
```

Comment:
Better to check also tlsconfig != Nil in order to avoid a nil pointer error in runtime

Reply:
I am not sure. It would be an example defensive programming. I'd rather prefer it to crash and make visible our semantics are not consistent. The tests would fail anyway if the semantics are changed.

Reply:
It's not for tests, but it's for server embedding.